docs/source/control_status_registers.rst

```diff
@@ -335,10 +335,31 @@ Detailed:
 | 31:0        | RW        | Number of iteration of HWLoop 0/1.        |
 +-------------+-----------+-------------------------------------------+
 
+.. _csr-uhartid:
+
+User Hardware Thread ID (``uhartid``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CSR Address: 0xCC8 (only present if ``PULP_XPULP`` = 1)
```

Comment:
similarly as above, do we need this only when PULP SECURE is 1?

Reply:
Agreed.